src/client/GnosisSafeClient.ts

```diff
@@ -145,16 +103,9 @@ class GnosisSafeClient {
   ): Promise<SafeBalanceResponse> {
     const address = ethers.utils.getAddress(gnosisSafe);
 
-    return await getBalances(
-      this.config.gateway,
-      network,
-      address,
-      selectedCurrency,
-      {
-        exclude_spam: true,
-        trusted: false,
-      }
-    );
+    const url = `https://zero-service-gateway.azure-api.net/gnosis/${network}/safes/${address}/balances/${selectedCurrency}?exclude_spam=true&trusted=false`;
+    const res = await fetch(url);
+    return await res.json();
```

Comment:
Can we wrap this (and any other api call we make here) in a try/catch and fall back to the gnosis safe sdk method if the zero-service-gateway fails?

Reply:
Good point, thanks, I will fix it